indico/modules/events/sessions/templates/display/session_display.html

```diff
@@ -131,7 +132,7 @@ <h3>{% trans %}Description{% endtrans %}</h3>
                     {%- trans %}Contribution list{% endtrans -%}
                 </a>
             {% endif %}
-            {% if not sess.is_poster %}
+            {% if not is_poster %}
```

Comment:
keep this as it was before and add a property to Session that contains the logic above